scripts/docker.sh

```diff
@@ -0,0 +1,4 @@
+#!/bin/bash
+
+npm install
+webpack-dev-server --compress --watch-poll

```

Comment:
This'll work, but it might be cleaner to have the docker start pass an env var, and let webpack.config.js read the env var when deciding whether to add the key to config.